### PR TITLE
chore: add argument verbose to print init_etcd debug info

### LINKED ISSUE #3242: chore: support option `verbose` to debug init_etcd
After #2101 got merged, the original option to debug init_etcd would be [removed](https://github.com/apache/apisix/pull/2101/files#diff-20dafebfcf815abe8b79cd8cc0b2483da24076105fa73b0d84682608b51e6b13L249-L251). https://github.com/apache/apisix/blob/7b5d2eae26466935c8be622b443ae23e59dbaa3b/apisix/cli/etcd.lua#L249-L251

As discussed in #2624, it would be a good idea to add another option `verbose` to achieve the same function. And we could still use pkg `argparse` to achieve it ([some sample codes](https://github.com/apache/apisix/pull/2101/files#diff-228a51571b53709ad043d1c6c14fc8e508c248a84f8014fe548b5d259cedd5ddR395-R409)).

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Signed-off-by: yiyiyimu <wosoyoung@gmail.com>

### What this PR does / why we need it:
fix #3242

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — APPROVED** [review]

**Comment by Firstsawyou** [comment]
LGTM.

**Review by membphis — COMMENTED** [review]
more doc? it is useful for the user or developer to know this feature.

**Comment by Yiyiyimu** [comment]
Hi @membphis it's actually an existing feature, that was temporarily removed in #2101. And this PR is to re-add it.
Actually, I'm not sure about the meaning of printing init_etcd debug info, like:
```
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"13","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"14","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"15","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"16","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"17","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"18","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"19","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"20","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"21","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"22","raft_term":"2"}}
{"header":{"cluster_id":"14841639068965178418","member_id":"10276657743932975437","revision":"23","raft_term":"2"}}
```
I don't know when do we need it

**Comment by membphis** [comment]
> Hi @membphis it's actually an existing feature, that was temporarily removed in #2101. And this PR is to re-add it.

thanks you, got it.

> Actually, I'm not sure about the meaning of printing etcd response when init each dir in init_etcd, like:

I do know the meaning too. How can we get those messages? I tried `./bin/apisix init_etcd`, I do not get it.

image

**Comment by Yiyiyimu** [comment]
@membphis you can try `apisix start --verbose` in this branch

File: apisix/cli/ops.lua [hunk 1]
```diff
@@ -450,20 +450,20 @@ local function start(env, ...)
     local parser = argparse()
     parser:argument("_", "Placeholder")
     parser:option("-c --config", "location of customized config.yaml")
+    parser:flag("--verbose", "show init_etcd debug information")
```
Comment (tokers) [inline apisix/cli/ops.lua:453]: I think `--verbose` can be designed not only for `init_etcd` but let APISIX cli works in verbose mode. So maybe "verbose mode" in description is better.

Comment (Yiyiyimu) [reply apisix/cli/ops.lua:453]: That is indeed a good idea I think. But I go through APISIX cli and for me I could not find many places to add more log. Maybe we could leave it here, and wait if anyone need to add some log.

Comment (tokers) [reply apisix/cli/ops.lua:454]: OK, got you.

**Review by tokers — APPROVED** [review]

**Comment by membphis** [comment]
> @membphis you can try `apisix start --verbose` in this branch

wow, I got them. I think we can delete them, they are unreadable.
we should print more meaningful message.

@Yiyiyimu 

image

**Comment by Yiyiyimu** [comment]
> wow, I got them. I think we can delete them, they are unreadable.
> we should print more meaningful message.

Got it. Would fix this problem next time when finding useful stuff to print.

**Comment by membphis** [comment]
you can create a new issue about this, todo things

**Comment by Yiyiyimu** [comment]
> you can create a new issue about this, todo things

Thanks for remind! Issue created as #3523

**Pull request merged**